easybuild/easyblocks/a/aomp.py

```diff
@@ -71,7 +70,6 @@ def configure_step(self):
             # Use the commandline / easybuild config option if given, else use
             # the value from the EC (as a default)
             cuda_cc = build_option('cuda_compute_capabilities')
-            cuda_cc = cuda_cc or self.cfg['cuda_compute_capabilities']
```

Comment:
You should still allow for cuda_compute_capabilities to be specified in the easyconfig, i.e. this line should not be removed.
See tensorflow.py line 551 for a example on how to use it.

Reply:
I thought you wanted me to remove the EB capability of defining `cuda_compute_capabilities` and that it had to be defined in the `extra_vars` option.

I have added back in the capability to either use the command line parameter or variable defined in EB (copied from TF EB).

Reply:
No, it was just that  you don't need to specify it in extra_vars since it is made available there from the framework.